migration/orm.go

```diff
@@ -63,13 +65,117 @@ func (svb Bucket) Save(db weave.KVStore, obj orm.Object) error {
 }
 
 func (svb Bucket) migrate(db weave.ReadOnlyKVStore, obj orm.Object) error {
-	m, ok := obj.Value().(Migratable)
+	return migrate(svb.migrations, svb.schema, svb.packageName, db, obj.Value())
+}
+
+func (svb Bucket) WithIndex(name string, indexer orm.Indexer, unique bool) orm.Bucket {
```

Comment:
Yes, very important to add wrapper here.

I was surprised by modifying the parameter, then realized it wasn't a pointer, so safely working on a copy. Nice command of Go details. :smile: